carma-messenger-config/cave_in_a_box/GlobalParamsOverride.yaml

```diff
@@ -0,0 +1,45 @@
+# This file allows easier configurations and overrides for individual nodes in carma-messenger
+# Package and node structure example is provided as a commented out code
+# Any parameters here will override the default parameters in the respective package's yaml files
+# For brevity the parameter descriptions are not provided here. The descriptions are located
+# in each package's parameters yaml file
+# Only the parameters that need to be overridden should be included in this file, rest will
+# use the default values in the respective package's yaml files
+# NOTE: Be sure to include the node name here, which may not be same as the package name
+# NOTE: SubsystemControllers has its own override file for easier organization
```

Comment:
This note on "SubsystemControllers" can be removed here and in other GlobalParamsOverride.yaml files since it is not used in carma-messenger

Reply:
Removed